### PR TITLE
Week3

### PR DESCRIPTION
**Pull request by butolinka**
week three, express

### CODE REVIEW

File: 02-express-tutorial/app.js [hunk 1]
```diff
@@ -1 +1,50 @@
...
+
+app.listen(3000, () => {
+    console.log('server is on');
+});
+app.use(express.static("./public"));
+app.get('/api/v1/test', (req, res)=>{
+    res.json({ message: "It worked!" });
+});
+app.get('/api/v1/products', (req, res)=>{
+    res.json({products });
```
Comment (jsuoz121) [inline 02-express-tutorial/app.js]: I don't think you should need the brackets here around products, but if you tested and it worked not a blocker!

File: 02-express-tutorial/app.js [hunk 2]
```diff
@@ -1 +1,50 @@
...
+
+    if(search){
+        sortedProducts = sortedProducts.filter((product)=>{
+            return product.name.startsWith(search);
+        });
+    }
+    if (limit){
+        sortedProducts =  sortedProducts.slice(0, Number(limit));
+    }
+    if (price){
```
Comment (jsuoz121) [inline 02-express-tutorial/app.js]: Better to do minPrice and/or maxPrice to clarify for users, unless you're going to allow `price<=<your number>` as the query parameter...which is more advanced than this course. Not blocking, just a comment for the future!

**Review by jsuoz121 — APPROVED** [review]
Looks like your commits from week2 are in here as well. Remember to merge your previous week into main! Let me know if you run into issues with the merge, and in the future just make sure you check out and pull `main` before creating the branch for each week's work. Good job with the express assignment

File: 02-express-tutorial/app.js (hunk 1 above)
Comment (butolinka) [reply 02-express-tutorial/app.js]: it worked, but if it was not correct I will fixed it. Thank you!)

**Pull request merged**